.travis.yml

```diff
@@ -1,14 +1,18 @@
----
 os: linux
 dist: bionic
 
 jobs:
   include:
-    - language: python
-      python: 3.8
-      cache: pip
-      install: "pip install -r requirements.txt"
-      script: sh run_tests.sh
+     - language: python
+
+python:
+  - "3.8"
+
+# command to install dependencies
+install: "pipenv install"
+
+# command to run tests
+script: sh run_tests.sh
```

Comment:
The Travis config appears to have reverted to an older status. This Pull Request needs to be rebased on the latest changes in `master`. Once the Travis config is fixed up, we can merge this and the docs PR next.